README.md

```diff
@@ -28,16 +28,17 @@ You may need the following: `install.packages("xaringan", "fontawesome")`
 
 ### Name your rmd file index.rmd
 
-This will mean it knits to index.html, which GH will host easily on GH pages. 
+This will mean it knits to index.html, which GH will host easily on GH pages.
 
 ### Accessibility
 
-Be sure to include informative alt text for every image. End alt text with punctuation (like a period), since that will make a screen reader read it more naturally. 
+Be sure to include informative alt text for every image. End alt text with punctuation (like a period), since that will make a screen reader read it more naturally.
 
 ### Slide notes
 
-At the bottom of a slide, type `???` and a new line to begin the slide notes. These will show up in presenter mode. Note that you can't have spaces after the ?'s, just the newline. If you provide live coding instructions in the slide notes, use `>` to distinguish that from speaking notes. 
+At the bottom of a slide, type `???` and a new line to begin the slide notes. These will show up in presenter mode. Note that you can't have spaces after the ?'s, just the newline. If you provide live coding instructions in the slide notes, use `>` to distinguish that from speaking notes.
 
+If you're using incremental slides, keep in mind that the presenters notes will behave in the same way as the content on the slides. That is, notes will only begin appearing in the slide section that you've written them, and will persist throughout the rest of the slide after that. (see [incremental slides](#incremental-slides) for more detail) Therefore, if you want your notes to display for the entire duration of the slide, you must add the notes prior to the first incremental divider.  before the first incremental divider will display on the screen for the duration of the slide, while notes written at the very bottom of the slide will only display once all slide content is on the screen. You can also add to your notes incrementally, if you prefer, by adding the additional notes that you'd like to display after each incremental addition to the slide content. If you put your presenters notes before the first divider, they will show up for the entire slide. If you write the notes in at the very bottom of the slide, they will only show up once all of the slide content has shown up on the screen. If you want to incrementally add to the notes
```

Comment:
I think you've got some missing words here ("Therefore, if you want your notes to display for the entire duration of the slide, you must add the notes prior to the first incremental divider.  before the first incremental divider will display on the screen for the duration of the slide, while notes written at the very bottom of the slide will only display once all slide content is on the screen."). I also think we probably want to delete my note immediately after this line, saying incremental slide notes only show up at the end, since you figured out how to make that work so much better (yay!). Otherwise, this looks great! Thanks for updating the docs!

Reply:
Oops -- I did some drafting in the document and clearly forgot to remove all the excess once I liked the way I had worded it. 

Thanks for catching this -- I'll go clean it up now!